From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Sun, 3 Jul 2016 15:18:29 -0400
Subject: Use Adwaita icons and cursors

Adwaita icon and cursor themes are already installed as dependencies of
GTK+3. I believe MATE's theme chooser (mate-appearance-properties) is the
only thing that uses these defaults and it shows an ugly warning banner if
any of the defaults aren't installed.
--- a/dark/index.theme
+++ b/dark/index.theme
@@ -7,6 +7,6 @@
 [X-GNOME-Metatheme]
 GtkTheme=Greybird-dark
 MetacityTheme=Greybird-dark
-IconTheme=elementary-xfce-dark
-CursorTheme=DMZ-Black
+IconTheme=Adwaita
+CursorTheme=Adwaita
 ButtonLayout=:minimize,maximize,close
--- a/light/index.theme
+++ b/light/index.theme
@@ -7,6 +7,6 @@
 [X-GNOME-Metatheme]
 GtkTheme=Greybird
 MetacityTheme=Greybird
-IconTheme=elementary-xfce-dark
-CursorTheme=DMZ-Black
+IconTheme=Adwaita
+CursorTheme=Adwaita
 ButtonLayout=:minimize,maximize,close
